src/storage_pool.cc

```diff
@@ -116,7 +117,16 @@ NAN_METHOD(StoragePool::LookupByUUID)
 NAN_METHOD(StoragePool::LookupByVolume)
 {
   Nan::HandleScope scope;
-  return;
+  auto hv = ObjectWrap::Unwrap<Hypervisor>(info.This())->handle_;
+  auto volume = ObjectWrap::Unwrap<StorageVolume>(info[0]->ToObject())->handle_;
+  
+  Worker::Queue(info[1], [=](Worker::SetOnFinishedHandler onFinished) {
```

Comment:
Hah, my opinion is totally opposite, I think the `Worker::Queue` should get the whole `info` object and just extract the last argument on its own. There is so much redundant code all over the place that little implicit behavior would make things so much easier and more error-proof. :) In other places you can see I was doing `info[info.Length() - 1]`. Can I convince you? :-)